test/atm/global-workflow/jcb-prototype_3dvar.yaml.j2

```diff
@@ -17,7 +17,7 @@ atmosphere_obsdatain_suffix: ".{{ current_cycle | to_YMDH }}.nc"
 
 # Testing things
 # --------------
-{% if DO_TEST_MODE and RUN == 'gdas'  %}
+{% if DO_TEST_MODE == true and RUN == 'gdas' and current_cycle | to_YMDH == '2021032318' %}
```

Comment:
suggestion for future, what about making the test reference file names include RUN and cycle times?

Reply:
Good idea